addons/danger/settings.inc.sqf

```diff
@@ -108,6 +108,70 @@ _curCat = LSTRING(Settings_GeneralCat);
     1
 ] call CBA_fnc_addSetting;
 
+addMissionEventHandler ["GroupCreated", {
+    params ["_group"];
+    [
+        {
+            if (count (units _this) > 0) then {
+                if (GVAR(autoAddDynamicReinforcement)) then {_this setVariable [QGVAR(enableGroupReinforce), true, true];};
+                if !(GVAR(disableAttackOverride)) then {_this enableAttack false;};
+                if (GVAR(disableFleeingOverride)) then {_this allowFleeing 0;};
+            };
+        },
+        _group,
+        3
+    ] call CBA_fnc_waitAndExecute;
+}];
+
+// auto disable enableAttack on groups
+[
+    QGVAR(disableAttackOverride),
+    "CHECKBOX",
+    [LSTRING(Settings_AttackOverride), LSTRING(Settings_AttackOverride_ToolTip)],
+    [COMPONENT_NAME, _curCat],
+    false,
+    false, {
+        params ["_value"];
+        if (_value) exitWith {};
+        if (!_value && time > 0) then {
+            {_x enableAttack false;} forEach (allGroups select {local (leader _x)});
+        };
+    }
+] call CBA_fnc_addSetting;
+
+// auto disable allowFleeing on groups
+[
+    QGVAR(disableFleeingOverride),
```

Comment:
This and the next setting need a XEH initPost eventhandler instead. People spawn in units all the time, which will not get caught by this.